### PR TITLE
LPS-161106 Fix bug that causes failures in ObjectField integration test

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-161106

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
@p-albuquerque I checked and the failures are not related to your changes. After making the suggested change, feel free to manually forward this PR to Brian.
Be sure to add a comment to explain why we need this change.

Thanks!

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -796,6 +796,22 @@ private ObjectField _deleteObjectField(ObjectField objectField)
...
+			ObjectFieldSetting objectFieldSetting =
+				_objectFieldSettingPersistence.fetchByOFI_N(
+					objectField.getObjectFieldId(), "fileSource");
+
+			String objectFieldSettingFileSource = objectFieldSetting.getValue();
+
+			if (Objects.equals(objectFieldSettingFileSource, "userComputer")) {
+				_deleteFileEntries(
+					objectField.getObjectDefinitionId(), objectField.getName());
+			}
```
[the remark this answers is not in the document]
Comment (p-albuquerque) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:812]: got it

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/124191

**Pull request closed without merging**